vertx-web/src/main/java/io/vertx/ext/web/handler/BodyHandler.java

```diff
@@ -66,6 +66,16 @@ static BodyHandler create() {
     return new BodyHandlerImpl();
   }
 
+  /**
+   * Create a body handler setting if it should handle file uploads
+   *
+   * @param ignorefilesUpload true if files upload is allowed
```

Comment:
Ah. You should still update this comment.

Reply:
Correct 👍